packaging/macosx/gtk-mac-bundler-0.7.4.patch

```diff
@@ -21,7 +21,7 @@ diff -uNr gtk-mac-bundler-0.7.4-orig/bundler/bundler.py gtk-mac-bundler-0.7.4/bu
          def name_filter(filename):
 @@ -585,14 +587,21 @@
              if program.name == "" or program.name == None:
-                 raise "No program name to tranlate!"
```

Comment:
Changing patch files is probably breaking them.

Reply:
apologies. this one `got passed the goalie`. Will remove and repush.